src/cares_wrap.cc

```diff
@@ -69,6 +69,19 @@ using v8::Value;
 
 namespace {
 
+inline uint16_t cares_get_16bit(const unsigned char* p) {
```

Comment:
@addaleax, @indutny. So shall I still have to change `cares_get_16bit` to `ntohs`?